ts/features/fci/hooks/useFciCheckService.tsx

```diff
@@ -67,6 +68,7 @@ export const useFciCheckService = () => {
               })
             );
           }
+          trackFciUxConversion();
           dispatch(fciStartSigningRequest());
           dismiss();
```

Comment:
Same as above about "tracking" and "dispatching". What do you think to have a standard also for "dismissing" and "dispatching"?